ui/pages/home/home.component.js

```diff
@@ -598,6 +691,11 @@ export default class Home extends PureComponent {
         />
         <div className="home__container">
           {showWhatsNew ? <WhatsNewPopup onClose={hideWhatsNewPopup} /> : null}
+          {process.env.TRANSACTION_SECURITY_PROVIDER &&
+          showWhatsNew &&
+          !openSeaTransactionSecurityProviderPopoverHasBeenShown
+            ? this.renderOpenSeaProviderPopover()
+            : null}
```

Comment:
Conditions above can be included into new component to render security provider. ref: https://github.com/MetaMask/metamask-extension/pull/16831/files#r1072384314

Reply:
Fixed here: b693772adac938c037cc398851c79bc16a7c0610